### PR TITLE
aribas 1.64 (new formula)

### PR DESCRIPTION
**Pull request by dtrebbien**

### CODE REVIEW

File: aribas.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          k5 = 327843840;
+      var
+          A, n: integer;
+          S: integer;
+      begin
+          A := zz*k1;
+          S := A * k2;
+          n := 1;
+          while A > 0 do
+                  A := A * ((6*n-5)*(6*n-3)*(6*n-1));
```
Comment (dpo) [inline aribas.rb]: Seems like the indentation needs to be fixed here.

File: aribas.rb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          x := isqrt(zz*zz*k3)*k6;
+          x := (zz * x) div Saux(zz);
+          return (x div 2**16);
+      end;
+
+      pi_chud(50).
+
+      exit.
+    EOS
+    system "#{bin}/aribas pi.ari | grep -q 3_14159_26535_89793_23846_26433_83279_50288_41971_69399_37510"
```
Comment (dpo) [inline aribas.rb]: Great test!

File: aribas.rb (hunk 1 above)
Comment (dtrebbien) [inline aribas.rb]: Yes, you're right.  New commit uploaded.

File: aribas.rb (hunk 2 above)
Comment (dtrebbien) [inline aribas.rb]: Thanks :smile:

**Pull request closed without merging**